python/across_api/base/footprint.py

```diff
@@ -0,0 +1,43 @@
+# Copyright © 2023 United States Government as represented by the
+# Administrator of the National Aeronautics and Space Administration.
+# All Rights Reserved.
+
+
+from astropy import units as u  # type: ignore[import]
+from astropy.coordinates.representation import (  # type: ignore[import]
+    CartesianRepresentation,
+    UnitSphericalRepresentation,
+)
+from astropy.coordinates.matrix_utilities import rotation_matrix  # type: ignore[import]
+
+
+class Footprint:
+    polygon: list = []
```

Comment:
Store polygon as a frame-free Representation rather than as a list.

```suggestion
    polygon: UnitSphericalRepresentation
```

Reply:
Implemented in recent push